src/assets/i18n/pt-BR.json5

```diff
@@ -6486,175 +6015,137 @@
-  // TODO New key - Add a translation
   "submission.import-external.source.sherpaJournal": "SHERPA Journals",
 
   // "submission.import-external.source.sherpaJournalIssn": "SHERPA Journals by ISSN",
-  // TODO New key - Add a translation
   "submission.import-external.source.sherpaJournalIssn": "SHERPA Journals by ISSN",
 
   // "submission.import-external.source.sherpaPublisher": "SHERPA Publishers",
-  // TODO New key - Add a translation
   "submission.import-external.source.sherpaPublisher": "SHERPA Publishers",
 
   // "submission.import-external.source.openAIREFunding": "Funding OpenAIRE API",
-  // TODO New key - Add a translation
   "submission.import-external.source.openAIREFunding": "Funding OpenAIRE API",
 
   // "submission.import-external.source.orcid": "ORCID",
-  // TODO New key - Add a translation
   "submission.import-external.source.orcid": "ORCID",
 
   // "submission.import-external.source.pubmed": "Pubmed",
-  // TODO New key - Add a translation
   "submission.import-external.source.pubmed": "Pubmed",
 
   // "submission.import-external.source.pubmedeu": "Pubmed Europe",
-  // TODO New key - Add a translation
   "submission.import-external.source.pubmedeu": "Pubmed Europe",
 
   // "submission.import-external.source.lcname": "Library of Congress Names",
-  // TODO New key - Add a translation
   "submission.import-external.source.lcname": "Library of Congress Names",
 
   // "submission.import-external.preview.title": "Item Preview",
-  // TODO New key - Add a translation
-  "submission.import-external.preview.title": "Item Preview",
+  "submission.import-external.preview.title": "Item Previsão",
 
   // "submission.import-external.preview.title.Publication": "Publication Preview",
-  // TODO New key - Add a translation
-  "submission.import-external.preview.title.Publication": "Publication Preview",
+  "submission.import-external.preview.title.Publication": "Publicação Previsão",
 
   // "submission.import-external.preview.title.none": "Item Preview",
-  // TODO New key - Add a translation
-  "submission.import-external.preview.title.none": "Item Preview",
+  "submission.import-external.preview.title.none": "Item Previsão",
 
   // "submission.import-external.preview.title.Journal": "Journal Preview",
   // TODO New key - Add a translation
   "submission.import-external.preview.title.Journal": "Journal Preview",
 
   // "submission.import-external.preview.title.OrgUnit": "Organizational Unit Preview",
-  // TODO New key - Add a translation
-  "submission.import-external.preview.title.OrgUnit": "Organizational Unit Preview",
+  "submission.import-external.preview.title.OrgUnit": "Unidade Organizacional Previsão",
 
   // "submission.import-external.preview.title.Person": "Person Preview",
-  // TODO New key - Add a translation
-  "submission.import-external.preview.title.Person": "Person Preview",
+  "submission.import-external.preview.title.Person": "Pessoa Previsão",
```

Comment:
Perhaps it would be better to have:
`Pré-visualização de Pessoa`

Reply:
ok